backends/bevy_picking_egui/Cargo.toml

```diff
@@ -13,7 +13,7 @@ resolver = "2"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-bevy = { version = "0.10", default-features = false }
+bevy = { version = "0.11", default-features = false }
 bevy_egui = "0.20"
```

Comment:
bevy_egui needs to be updated to 0.21 since bevy_egui 0.20 only supports bevy 0.10.

BTW: bevy_egui 0.21 uses egui 0.22 while bevy_egui 0.20 uses egui 0.21. Just adding this here since the version numbers are a little confusing.